### ISSUE TITLE
[Suegestion] a way to generate steam from fusion reactor When cooling it

### ISSUE THREAD

**Issue by Danishboi** [turbodiesel4598/NuclearCraft#492]
I don't know if it has been talked about before or it's on the to-do list, but it will be cool if when you cool down the fusion reactor it heats up the fluid and then it can be used in the heat exchanger to make steam to make additional power, or to make so that you have to maintain the active cooler by removing hot version of the fluid and cooling it down as you do with the molten salt reactor.

**Comment by turbodiesel4598** [comment]
Yes, this is already on the to-do list. The fusion reactor (and fission reactor, for this matter) is being rewritten entirely and will generate steam instead of power diretly ;)

**Comment by Danishboi** [comment]
ok nice

**Comment by Lykrast** [comment]
> The fusion reactor (and fission reactor, for this matter) is being rewritten entirely and will generate steam instead of power diretly ;)

Would that mean a turbine will always be required to get power out of them? If yes will there be some "crude turbine" for quick mid game setups for them?

**Comment by tomdodd4598** [comment]
@Lykrast There will be a low-tech pebble-bed reactor that will generate power directly instead of steam. Otherwise yes, at least one turbine stage will be needed. The way the reactors are changing, particularly the stellarator fusion, will mean that the turbine will be the last thing to worry about getting set up ;)

EDIT: this plan was changed - PBRs are now in, but they also require a turbine. Ultimately NCO is now a slightly later-game mod than pre-overhaul NC was.
